spp_custom/__manifest__.py

```diff
@@ -5,7 +5,7 @@
 {
     "name": "SPP Customizations",
     "version": "12.0.1.1.0",
-    "depends": ["beesdoo_base", "beesdoo_product"],
+    "depends": ["beesdoo_base", "product"],
```

Comment:
Can you do the dependency list like so?

```python
"depends": [
    "beesdoo_base",
    "product",
],
```

It reduces conflicts like the one we had.

Also, are you sure `beesdoo_base` is a correct dependency here? It's an emptied module. Can we change it instead to some of the dependencies of `beesdoo_base` itself?

Reply:
removed beesdoo_base
reformatted the dependencies but precommit does not let me... so I forced it. Weird...

Reply:
You need to include a trailing comma after the second item. See `"product",`